crates/sui-graphql-rpc/src/types/error.rs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use async_graphql::{Error, ErrorExtensions};
+
+#[derive(Debug, thiserror::Error)]
+pub enum CustomError {
```

Comment:
What's the significance of the term `CustomError` vs just `Error`?  If it's to denote that it's this project's error type, the pattern I'm used to is to just call it `Error` and rely on the module name to distinguish.